### ISSUE TITLE
F2 did not center to board

### ISSUE THREAD

**Issue by ScottinNH** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#485]
Here tell us what you thought was meant to happen and then how TaleSpire vomited on itself
   Anticpated behavior:   I had made a bunch of temp/test structures, built a final version, and then deleted all temp/test structures.  Could not see my final structure so pressed F2 center my in my board. 

  Observed issue:  After deleting I could not find my final structure, so I pressed F2 to center on board.  The screen did move but, did not center on the only structure not deleted on the board.  The only way I was able to find my structure was to make a search grid, and I finally found the structure.

**To Reproduce**
1. Make a squares(~25x25) space ~10 apart in a 9 x 9 grid.  S
2. Press F2 to center on board
3. Does not center on grid
4. Delete all but block
5. F2 does not center on remaining block
6. Repeatable

Suggestion:  In the far future.....  When map get very large, a low res top down mini-map navigator  would very handy.

**Comment by ScottinNH** [comment]
After more consideration I believe this to be operator error.  I did not understand that it was returning to the center of TaleSpire defined center of the board.   I was thinking what ever the objects the user place on the board was the board.  So since my objects where not near the center F2 was not helpful to me.
I did try Cut Scene Mode, grabbing a shot and then using that as a short back to desired area.  But some of my grabbed shots do not show up in later sessions.
